app/views/groups/_update_progress_bar.js.erb

```diff
@@ -0,0 +1,13 @@
+<script>
```

Comment:
Don't include the `script` tag here. Instead put it where the partial is being included.